README.md

```diff
@@ -1,77 +1,98 @@
-| license              | License the plugin is released under.                                                                            |          |
-| permissions          | List of permission the plugin is asking (`keypresses`, `use_eud`, `ext_api`).                                    | x        |
-| dependencies         | List of plugin's dependencies.                                                                                   | x        |
-| optionalDependencies | List of plugin's optional dependencies. Will only affect load order to ensure the plugin is loaded before yours. | x        |
-| appMode              | Where the plugin should be injected. Can be `app`, `overlay`, or `both` (default to `app`).                      | x        |
-
-## 2. Follow Discord and other services ToS
+### 2. Follow Discord and other services ToS
 
 Even if Powercord is by nature against Discord ToS, it doesn't mean that you're allowed to do everything you want. For
 example scraping End User Data without consent is forbidden.
 
-## 3. No circumvention of Discord permissions
+### 3. No circumvention of Discord permissions
 
-Do not circumvent Discord permissions as well as attempt to bypass something restricted by the Discord client. This can be a potential security breach or invasion of privacy (e.g. showing hidden channels is an invasion of staff privacy, storing
-deleted messages forever is an invasion of author's privacy (and forbidden by Discord ToS).
+Do not circumvent Discord permissions as well as attempt to bypass something restricted by the Discord client. This can
+be a potential security breach or invasion of privacy (e.g. showing hidden channels is an invasion of staff privacy,
+storing deleted messages forever is an invasion of author's privacy (and forbidden by Discord ToS)).
 
-## 4. Meet a certain standard performance wise
+### 4. Meet a certain standard performance wise
 
 Plugins that are heavy in terms of resource usage are not great for the user experience. Slower computers may not
 be able to run smoothly if your plugin is doing resource-heavy tasks, for example having mutation observers.
 
-If you do need to perform resource-heavy tasks, try using a [web worker](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers)
+If you do need to perform resource-heavy tasks, try using a [web worker](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers).
 
-## 5. Do not write harmful plugins
+### 5. Do not write harmful plugins
 
-Harmful plugins that abuse Powercord APIs or Discord internals to do malicious things like spambots, keyloggers
-or anything else are strictly prohibited.
+Harmful plugins that abuse Powercord APIs or Discord internals to do malicious things (including but not limited to
+spambots, keyloggers, scrapers, ...) are strictly prohibited.
 
-## 6. Do not have bad privacy practices
+### 6. Respect users privacy
```

Comment:
Respect users' privacy would sound better (imo), or maybe
Respect our users' privacy

Reply:
user's or user's (not sure which)